android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -4387,6 +4308,49 @@ public static void handleAppBarButtonPressed(ViewerComponent component, MenuItem
         }
     }
 
+    public static void handleCustomAppBarButton(ViewerComponent component) {
```

Comment:
Why does this need to be public?

Reply:
change to private since only classes inside PluginUtils accesses this function